public/v4/apps/mysql-backup.yml

```diff
@@ -0,0 +1,46 @@
+captainVersion: 4
+services:
+    $$cap_appname-db:
+        image: databack/mysql-backup:$$cap_mysql_backup_version
+        restart: always
+        environment:
+            DB_SERVER: $$cap_db_server
+            DB_PORT: $$cap_db_port
+            DB_USER: $$cap_db_user
+            DB_PASS: $$cap_db_password
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_mysql_backup_version
+          label: Mysql Backup Version
+          defaultValue: 'v0.11.0'
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/databack/mysql-backup/tags
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_db_server
+          label: Database Server
+          defaultValue: 'srv-captain--'
+          description: Database Server
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_db_port
+          label: Database Port
+          defaultValue: '3306'
+          description: Database Port
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_db_user
+          label: Database User
+          defaultValue: 'root'
+          description: Database User
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_db_password
+          label: Database Password
+          defaultValue: ''
+          description: Database Password
+          validRegex: /^([^\s^\/])+$/
+    instructions:
+        start: This installation just aims to create the initial container, you should at the end revise the documentation and set the extra environment parameters that fit your needs. Such as scheduling and to where you want to send your backups (AWS, SMB, etc)
```

Comment:
Add some link to the documentation. The name is very generic and it's not possible for the end user to find the proper env vars.

Reply:
The env vars/documentation here is the docker.io/hub one (https://hub.docker.com/r/databack/mysql-backup). Since there is so many options and the way it works I felt like if I was going to facilitate how people set we would need to deploy and maintain too many one-app-versions which in my opinion doesn't make sense and is messy.